datareduction/datareductionoperator.cpp

```diff
@@ -1423,7 +1423,6 @@ namespace DRO {
       const bool calculateNonthermal = true;
       //Calculate and save:
       PTensorOffDiagonalNonthermalCalculations( cell, calculateNonthermal, averageVX, averageVY, averageVZ, popID, PTensor );
-      const uint vectorSize = 3;
       //Input data into buffer
       const char* ptr = reinterpret_cast<const char*>(&PTensor);
       for (uint i = 0; i < 3*sizeof(Real); ++i) buffer[i] = ptr[i];
```

Comment:
Actually, for these I'd recommend we rather use vectorSize in this loop, because we can indeed have datareducers which have different length vectors.

Reply:
Alright, updated. vectorSize is now being used (and I properly formatted the loops with braces, too!)